spec/factories.rb

```diff
@@ -29,18 +33,33 @@
   end
 
   factory :move do
-    sequence(:name) { |n| "Sybil #{n}th" }
+    sequence(:name) { |n| "#{Faker::App.name} #{rand(1000)}th" }
```

Comment:
I have a feeling we could avoid strange test failures in the future by not using `rand(1000)` but instead `n`. By using `rand` it is technically possible for two moves to be generated with the same name, which would cause the validations to fail.